### PR TITLE
3207 fix projectile problem description

### PR DESCRIPTION
**Pull request by harmanpreet-sagar**
Fixes the Projectile Problem Statement as outlined in #3207. Changes have been made to stable code and the body of the Projectile example.

### CODE REVIEW

**Review by smiths — COMMENTED** [review]
Stable has been changed to properly reflect the requested change.

File: code/drasil-example/projectile/lib/Drasil/Projectile/Body.hs [hunk 1]
```diff
@@ -29,7 +29,7 @@ import Data.Drasil.Theories.Physics (accelerationTM, velocityTM)
 import Data.Drasil.TheoryConcepts (dataDefn, genDefn, inModel, thModel)
 
 import Drasil.Projectile.Assumptions (assumptions)
-import Drasil.Projectile.Concepts (concepts, landingPosNC,
+import Drasil.Projectile.Concepts (concepts,
   launcher, projectile, target)
```
[the remark this answers is not in the document]
Comment (harmanpreet-sagar) [reply code/drasil-example/projectile/lib/Drasil/Projectile/Body.hs:33]: For sure. I will make that change right now.

**Review by samm82 — APPROVED** [review]
Looks good to me! Just a note for future, try to use more descriptive commit messages; something like "reformat import statement" tells us more about what the change you made was which makes it easier if we need to look back at previous commits (which we often do).

**Review by JacquesCarette — APPROVED** [review]
Note that if you put "fixes #3207" in your commit message, then when I pull in the PR, it would close that issue automatically.

**Comment by harmanpreet-sagar** [comment]
Thank you for the feedback and the tip! I will keep them in mind the next time I make commits and PRs.

**Pull request merged**